### PR TITLE
Add LICENSE file and classifiers to setup.py

### PR DESCRIPTION
**Pull request by kitchoi**
This PR adds a license file (added it to the source distribution too) and the relevant mention in `setuptools.setup` metadata.

### CODE REVIEW

**Review by rahulporuri — APPROVED** [review]
LGTM.

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
+This software is OSI Certified Open Source Software.
+OSI Certified is a certification mark of the Open Source Initiative.
+
+Copyright (c) 2020, Enthought, Inc.
```
Comment (kitchoi) [inline LICENSE.txt:4]: Or 2018-2020 like the rest of the files in this project? Though I guess that range is copied from traits-future, which might not be the same as the lifespan of the copyright header checker.

Comment (rahulporuri) [reply LICENSE.txt:4]: sorry. i didn't notice the copyright in the rest of the modules. IIUC it should be 2020 everywhere.

Comment (kitchoi) [reply LICENSE.txt:4]: Cool. I will fix those in a separate PR then.

**Comment by kitchoi** [comment]
The webhook to Travis CI seems to have gone missing. Retrying...

**Pull request merged**